src/lava/magma/core/model/py/ports.py

```diff
@@ -20,7 +21,22 @@ def csp_ports(self) -> ty.List[AbstractCspPort]:
         pass
 
 
-class PyInPort(AbstractPyPort):
+class AbstractIOPyPort(AbstractPyPort):
```

Comment:
Naming slightly inconsistent with other PyPorts. Others are already called PyInPort, PyOutPort, PyRefPort. The Type of the port and the fact that it is a port are more closely bound than the the fact that there are all kinds of variants like Py, C, Nc, ... Thus it would be logical to call this one AbstractPyIOPort.

Reply:
Fixed